packages/portals/backend/sirius-components-collaborative-portals/src/main/java/org/eclipse/sirius/components/collaborative/portals/PortalEventProcessor.java

```diff
@@ -104,7 +104,6 @@ public void handle(One<IPayload> payloadSink, Many<ChangeDescription> changeDesc
                 IPortalEventHandler portalEventHandler = optionalPortalEventHandler.get();
                 PortalContext context = new PortalContext(this.representationSearchService, this.editingContext, this.currentPortal, portalInput);
                 portalEventHandler.handle(payloadSink, changeDescriptionSink, context);
-                context.getNextPortal().ifPresent(newPortal -> this.updatePortal(portalInput, newPortal));
```

Comment:
This seems to have broken something with the renaming of portals. If I understand things correctly, I suspect that this change AND the way change descriptions for representation renaming are performed, nothing is sent to the frontend if the portal itself is renamed. I'll investigate a bit further.